pkg/query/args.go

```diff
@@ -404,10 +405,30 @@ func (a *Args) Prepare(writers ...io.Writer) (*Statement, error) {
 
 	}
 
+	// set and validate the interfaces
+	if a.Ifaces == "" {
+		err := newArgsError(
+			"iface",
+			invalidInterfaceMsg,
+			&types.ParseError{
+				Description: "empty input",
```

Comment:
Maybe this could be more specific (as in: "no interface name specified") to be more clear to the recipient and to be in line with the "invalid interface name" message (language-wise)?